src/Microsoft.AspNet.Mvc.ApiExplorer/ApiResponseFormat.cs

```diff
@@ -3,7 +3,7 @@
 
 using Microsoft.Net.Http.Headers;
 
-namespace Microsoft.AspNet.Mvc.Description
+namespace Microsoft.AspNet.Mvc.ApiExplorer
```

Comment:
@Eilon thoughts on the namespace/naming of assembly? We called it `.Description` because that was the suffix in webapi.